addons/web/static/src/core/dropdown/dropdown.js

```diff
@@ -221,6 +224,10 @@ export class Dropdown extends Component {
         }
     }
 
+    setPopoverCloseOnClickAway(activeEl) {
+        return (target) => this.uiService.getActiveElementOf(target) === activeEl;
```

Comment:
Hello @abd-msyukyu-odoo, I'm not sure it's necessary to return a function inside the function. Unless there's a reason not to, I think it would look cleaner if we just write the function like this:
```js
setPopoverCloseOnClickAway(target, activeEl) {
        return this.uiService.getActiveElementOf(target) === activeEl;
}
```
and call the function directly in the closeOnClickAway option.